blog.araya.dev/posts/2021-02-16-css-cascade-layer.md

```diff
@@ -386,7 +386,7 @@ Layer を用いることで Specificity より先に Layer による優先度を
 @layer theme url("my-theme.css");
 
 @layer default {
-  button {
+  .button {
```

Comment:
```suggestion
  button {
```

If you are referring to the fact that Layer takes precedence over Specificity, then this change should be undone.